src/ConnectionTicket.h

```diff
@@ -0,0 +1,19 @@
+// Copyright © 2008-2022 Pioneer Developers. See AUTHORS.txt for details
+// Licensed under the terms of the GPL v3. See licenses/GPL-3.txt
+
+#pragma once
+
+#include "sigc++/connection.h"
+
+struct ConnectionTicket {
```

Comment:
I'd recommend a short doc comment on this struct for future developers, something like "Lifetime management utility for sigc::connection".

Reply:
Done!